### ISSUE TITLE
Drop scala :)

### ISSUE THREAD

**Issue by nixel2007** [nixel2007/intellij-lsp#4]
Pure java FTW :)

**Comment by nixel2007** [comment]
TODO: make scala2java roadmap

**Comment by NipunaRanasinghe** [comment]
If we are rewriting this using Java (I think we should :)), IMO we can do that separately (maybe in a separate repo)? WDYT?

**Comment by nixel2007** [comment]
I hope, that we can start with rewriting tests to java, then some parts of scala classes. Scala can use java classes (can't it?), so we can rewrite it partialy in long time.
I really don't want to use some other repo instead of feature branches for it. Sync problems and long-living-branch problems, you know.

**Comment by NipunaRanasinghe** [comment]
Yeah you are right. I meant if we could create a separate organization and start working on a fork there, it will be better for #5 and #6 as well. Any way we can move with after having some progress and considering the other's ideas as well :)

**Comment by cdietrich** [comment]
i am not too fond of scala. is more than ten years ago i really did a lot with it. having java 8 and lambdas gives already a lot of options

**Comment by nixel2007** [comment]
@NipunaRanasinghe, ah, got the point. I could create an "intellij-lsp" organinization and transfer this repo to it. Ok?

**Comment by NipunaRanasinghe** [comment]
@nixel2007 Exactly! :) IMO by that we could make these as more community-driven projects and probably gather JetBrains related language server clients into one place in the long run. Thanks for your effort on this!

**Comment by NipunaRanasinghe** [comment]
@nixel2007 One small suggestion, since we will be making this organization as the 'Home' for more jetbrains + language server developments in the long run, shall we name this organization in a more general sense ( maybe something like lsp4jetbrains ;) ) ?

**Comment by nixel2007** [comment]
@NipunaRanasinghe the platform is called "IntelliJ Platform", LSP is LSP... :) So i think we can leave this name for organization. We could try to rename the repo itself in more specific to "plugin" way. like intellij-lsp-plugin

**Comment by NipunaRanasinghe** [comment]
@nixel2007 +1 :)

**Comment by gayanper** [comment]
Guys i'm back after a while on Idea development again, well you have made great progress, so are we planning to move forward with this repo instead of the original ? May be we might need to publish the plugin with a different ID then. What do you guys think ?

**Comment by cdietrich** [comment]
@gayanper Guillaume seems to be back so i think we should join efforts at the original repo
